theme_graphene/views/customizations.xml

```diff
@@ -109,6 +113,15 @@
     <xpath expr="//div[hasclass('carousel-inner')]/div[3]" position="attributes">
         <attribute name="style" add="background-position: 50% 100%;" separator=";"/>
     </xpath>
+    <xpath expr="//div[hasclass('o_we_bg_filter')]" position="attributes">
+        <attribute name="class" add="bg-white-25" remove="bg-black-25" separator=" "/>
+    </xpath>
+    <xpath expr="(//div[hasclass('container')])[2]" position="before">
+        <div class="o_we_bg_filter bg-white-75" />
```

Comment:
I'm sorry but I really think that these combinations using `bg-white-75` over heavily detailed images with "thin black text" are very difficult to read  😕

| this PR | proposal |
|--------|--------|
| <img width="1194" alt="image" src="https://github.com/user-attachments/assets/c280c636-5a06-490e-b2e7-a22e7140965d"> | <img width="1188" alt="image" src="https://github.com/user-attachments/assets/0ec2abb9-954f-494c-85b5-b0ea55ce5b23"> |